aws/gpu_tree_shap/gpu_tree_shap.ipynb

```diff
@@ -0,0 +1,1232 @@
+{
```

Comment:
<ol><li>Mentioning AWS in the title might be helpful. Something like "GPU-Accelerated SHAP with XGBoost 1.3 on AWS"</li><li>I'd recommend adding Headings to divide sections, making it easier to navigate.</li><li>Add a few links for shap values and dataset so that the user can get more context.</li><li>"...XGBoost model with AWS SageMaker's.."</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>